geonode/harvesting/harvesters/base.py

```diff
@@ -206,19 +261,20 @@ def get_geonode_resource_defaults(
 
         defaults = {
             "owner": harvestable_resource.harvester.default_owner,
-            "uuid": str(resource_descriptor.uuid),
-            "abstract": resource_descriptor.identification.abstract,
-            "bbox_polygon": resource_descriptor.identification.spatial_extent,
-            "constraints_other": resource_descriptor.identification.other_constraints,
-            "created": resource_descriptor.date_stamp,
-            "data_quality_statement": resource_descriptor.data_quality,
-            "date": resource_descriptor.identification.date,
-            "date_type": resource_descriptor.identification.date_type,
-            "language": resource_descriptor.language,
-            "purpose": resource_descriptor.identification.purpose,
+            "uuid": str(harvested_info.resource_descriptor.uuid),
+            "abstract": harvested_info.resource_descriptor.identification.abstract,
+            "bbox_polygon": harvested_info.resource_descriptor.identification.spatial_extent,
+            "constraints_other": harvested_info.resource_descriptor.identification.other_constraints,
+            "created": harvested_info.resource_descriptor.date_stamp,
+            "data_quality_statement": harvested_info.resource_descriptor.data_quality,
+            "date": harvested_info.resource_descriptor.identification.date,
+            "date_type": harvested_info.resource_descriptor.identification.date_type,
+            "language": harvested_info.resource_descriptor.language,
+            "purpose": harvested_info.resource_descriptor.identification.purpose,
             "supplemental_information": (
-                resource_descriptor.identification.supplemental_information),
-            "title": resource_descriptor.identification.title
+                harvested_info.resource_descriptor.identification.supplemental_information),
+            "title": harvested_info.resource_descriptor.identification.title,
+            "files": harvested_info.copied_resources
```

Comment:
@afabiani This is where I am recording information about the copied files. The contents of `harvested_info.copied_resources` is a list populated with the paths returned by the `storage_manager` (if there were any files that have been copied that is) - this is later fed to `resource_manager` when calling its `create` or `update` methods, see the implementation of `update_geonode_resource()` on the base harvester worker class.